### PR TITLE
pest control: layout pc widgets & fix overlay ordering

### PR DESCRIPTION
**Pull request by raiyni**
Since runescape widgets aren't layoutable by default, they get consumed by RL widgets.

Preview:
peek 2018-09-05 20-36

### CODE REVIEW

**Comment by raiyni** [comment]
changed to use the recent layout code

**Comment by deathbeam** [comment]
Do you have screenshot?

File: runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java [hunk 1]
```diff
@@ -78,6 +78,9 @@
 	DIARY_QUEST_WIDGET_TITLE(WidgetID.DIARY_QUEST_GROUP_ID, WidgetID.Diary.DIARY_TITLE),
 	DIARY_QUEST_WIDGET_TEXT(WidgetID.DIARY_QUEST_GROUP_ID, WidgetID.Diary.DIARY_TEXT),
 
+	PEST_CONTROL_BOAT_INFO(WidgetID.PEST_CONTROL_BOAT_GROUP_ID, WidgetID.PestControlBoat.INFO),
+
```
Comment (deathbeam) [inline runelite-api/src/main/java/net/runelite/api/widgets/WidgetInfo.java]: Remove this empty line

File: runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java [hunk 2]
```diff
@@ -141,8 +142,15 @@
...
 
+	static class PestControlBoat
+	{
+		static final int INFO = 3;
+	}
+
 	static class PestControl
 	{
+		static final int INFO = 3;
+
```
Comment (deathbeam) [inline runelite-api/src/main/java/net/runelite/api/widgets/WidgetID.java]: Remove this empty line

**Pull request merged**